### DISCUSSION TITLE
What are the similarities/differences with foxythemes' jira-cli?

### DISCUSSION THREAD

**Discussion by darcyparker** [ankitpokhrel/jira-cli#260] [Q&A] [answered]
I see there is a similar project https://github.com/foxythemes/jira-cli (https://jiracli.com/#features) that doesn't seem to be actively supported anymore.

Has anyone compared the projects? Are there other jira-cli tools to consider too?

**Comment by ankitpokhrel** [comment, marked as answer]
@darcyparker there are lot of cli tools for jira out there. I haven't done any comparisons myself but I found [this comparison](https://yeti.sh/articles/jira-cli/) over the internet if that helps. The article states this jira-cli doesn't support `fixVersions` which won't be entirely true from the next release ;)

**Comment by darcyparker** [comment]
Very useful - thanks. I think it would be worth adding that to the README.
